### ISSUE TITLE
SRSClient results have the wrong timestamp

### ISSUE THREAD

**Issue by Cadair** [sunpy/sunpy#4922]
When the SRSClient downloads a tarball and extracts it, we should show in the results table and entry for every day that we will extract from the tarball, so if I search for 3 days the results should show me 3 day long records, which will correspond to the files when extracted from the tarball.

It seems this is not the case: https://github.com/sunpy/sunpy/pull/4789/files#r560468008

**Comment by hayesla** [comment]
maybe I'm missing something here - do you mean the results table of the search?

**Comment by samaloney** [comment]
yea me to I think #4904 does this 

https://github.com/sunpy/sunpy/pull/4904/files#diff-33ff99802afd99383a4aaae139689c87077764ee59e4035ab6981c51950b7367R153-R156

**Comment by samaloney** [comment]
So I think this can be closed now as #4904 also resolved this?

```python
query = Fido.search(a.Time('2018-04-01', '2018-05-15'), a.Instrument.srs_table)
query
<sunpy.net.fido_factory.UnifiedResponse object at 0x7f7b4d587e80>
Results from 1 Provider:

45 Results from the SRSClient:
       Start Time               End Time        Instrument Physobs Source Provider
----------------------- ----------------------- ---------- ------- ------ --------
2018-04-01 00:00:00.000 2018-04-01 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-04-02 00:00:00.000 2018-04-02 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-04-03 00:00:00.000 2018-04-03 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-04-04 00:00:00.000 2018-04-04 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-04-05 00:00:00.000 2018-04-05 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
                    ...                     ...        ...     ...    ...      ...
2018-05-11 00:00:00.000 2018-05-11 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-05-12 00:00:00.000 2018-05-12 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-05-13 00:00:00.000 2018-05-13 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-05-14 00:00:00.000 2018-05-14 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
2018-05-15 00:00:00.000 2018-05-15 23:59:59.999  SRS-TABLE     SRS   SWPC     NOAA
Length = 45 rows
```
